assessment.md

```diff
@@ -306,17 +306,51 @@ TODO: add description for each assessment level for this skill.
 ### Interactions
 
 * Interaction with the customers
-
-TODO: add description for each assessment level for this skill.
+- SERVICE
+- Social
+- Eager
+- Replace
+- Verbal Communication
+- Informative
+- Countability
+- Easy
+
+The objective for the interaction with a customer is supposed to be smooth and easy. Interacting with customers is one of the most important parts of software engineering. The acronym of "SERVICE" is a great way for us to see what attributes and tendencies lead to good service.
+
+- :flushed: : refuses to interact with and/or work with customers. Ignores specifics and details regarding the tool from the customer.
```

Comment:
Capitalize the sentence just like the rest of the assessment. There are also few spots below that are having the same issue.